src/routes/docs/Sidebar.svelte

```diff
@@ -112,6 +112,11 @@
                     href: '/docs/tooling/assistant',
                     icon: 'icon-sparkles'
                 },
+                {
+                    label: 'MCP',
+                    href: '/docs/tooling/mcp',
+                    icon: 'icon-sparkles'
```

Comment:
Added `icon-sparkles` temporarily because we'll need to add an MCP icon to Pink Icons @christyjacob4